app/models/package.js

```diff
@@ -104,6 +104,10 @@ export default Model.extend(cloudinaryImage, {
     return !res ? "" : res + " cm";
   }),
 
+  isDimensionPresent: computed("width", "height", "length", function() {
```

Comment:
```suggestion
  isDimensionPresent: and("width", "height", "length")
```
You can try this one.

Reply:
Okay got it